utils/core/src/serde/mod.rs

```diff
@@ -53,6 +53,90 @@ impl Serializable for () {
     fn write_into<W: ByteWriter>(&self, _target: &mut W) {}
 }
 
+impl<T1> Serializable for (T1,)
```

Comment:
minor nit: do we need to have a comma here after `T1`?

Reply:
yes:

> The syntax for tuple expressions is a parenthesized, comma separated list of expressions, called the tuple initializer operands. 1-ary tuple expressions require a comma after their tuple initializer operand to be disambiguated with a [parenthetical expression](https://doc.rust-lang.org/reference/expressions/grouped-expr.html).

https://doc.rust-lang.org/reference/expressions/tuple-expr.html